plugins/callback/foreman.py

```diff
@@ -292,17 +292,10 @@ def send_reports_proxy_host_report(self, stats):
                         "total": int(get_time() - self.start_time)
                     }
                 },
-                "status": {
-                    "applied": total['changed'],
-                    "failed": total['failures'] + total['unreachable'],
-                    "skipped": total['skipped'],
-                },
+                "summary": total,
```

Comment:
does that require any changes to the `report` plugin? or does it already understand this format?

Reply:
Yes, however, we pulled the plugin from 3.1 (ehm) and there is no stable release yet, so changes will be made. I can link a PR (not up yet), I am waiting for our community to review my final proposal of the mapping:

https://community.theforeman.org/t/new-config-report-summary-columns/26531

Reply:
Cool.

Btw, should the variable be renamed to `summary` too? It's the result of `summarize(host)` and you submit it as `summary` field…

Reply:
Good point, I removed the variable completely, there are no other uses anymore.